doktor 7675-7679

doktorsleepelss opened the pull request.

spektor56 commented:
https://spektor56.github.io/comma10kreviewer/?index=0&pr=2625
Dont think this in undrivable here.  Looks like a little ramp of asphalt up to the sidewalk.  Theres also a left turn lane that turns into that area.

pjlao307 commented:
This PR was missing 7675 so I updated the title and the spreadsheet to reflect this.

doktorsleepelss commented:
> This PR was missing 7675 so I updated the title and the spreadsheet to reflect this.

I added it to https://github.com/commaai/comma10k/pull/2627 since I didn't realize I missed it here.
